fields/InputField.vue

```diff
@@ -37,13 +37,21 @@
           @click="isPasswordShown = !isPasswordShown"
         >
           <icon
-            class="input-field__password-icon"
+            class="input-field__icon"
             :name="isPasswordShown ? ICON_NAMES.eye : ICON_NAMES.eyeOff"
           />
         </button>
+        <button
+          v-else-if="isRemovable"
```

Comment:
What does it mean? Is it a button to clear the field? If so, it should be called something else.

Reply:
done, now it is isClearable